test/controllers/api/capabilities_controller_test.rb

```diff
@@ -9,6 +9,10 @@ def test_routes
         { :path => "/api/capabilities", :method => :get },
         { :controller => "api/capabilities", :action => "show" }
       )
+      assert_routing(
+        { :path => "/api/capabilities.json", :method => :get },
+        { :controller => "api/capabilities", :action => "show", :format => "json" }
+      )
       assert_recognizes(
         { :controller => "api/capabilities", :action => "show" },
         { :path => "/api/0.6/capabilities", :method => :get }
```

Comment:
Should there be another `assert_recognizes` here for the JSON route?

Reply:
Yes it does need that so I've pushed a commit to add it...